operate/client/src/modules/components/Diagram/styled.ts

```diff
@@ -21,6 +21,10 @@ const DiagramCanvas = styled.div`
   left: 0;
   top: 0;
 
+  .op-highlighted.djs-shape .djs-visual > :nth-child(1) {
+    stroke: var(--cds-background-brand) !important;
+  }
```

Comment:
:bulb: This method is suggested here: https://github.com/bpmn-io/bpmn-js-examples/blob/main/colors/README.md#option-3-color-via-marker--css-styling